.buildkite/pipeline_perf.py

```diff
@@ -28,6 +28,8 @@
         "label": "📠 Network Latency",
         "test_path": "integration_tests/performance/test_network_ab.py::test_network_latency",
         "devtool_opts": "-c 1-10 -m 0",
+        # Triggers if delta is > 0.01ms (10µs) or default relative threshold (5%)
+        "ab_opts": "--relative-strength 0.010",
```

Comment:
Ahh, sorry, I misunderstood what you meant. This is still a percentage value. However, since the default is 0 and the argument is not used anywhere, you can just repurpose it into an "absolute delta" by changing line 284 from `and abs(result.statics) > baseline_mean * strength_tresh` to `abs(result.statistic) > strength_tresh` (and maybe the argument to `absolute-strength`)